csharp-basic-topics/Zip-files-in-NET/Zip-files-in-NET/Zip-files-in-NET.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <RootNamespace>Zip_files_in_NET</RootNamespace>

```

Comment:
Solutions/Projects should follow the PascalCase convention.

Reply:
Corrected